packages/platforms/browser/tests/backgrounding-listener.test.ts

```diff
@@ -29,50 +29,150 @@ class DocumentStub {
   }
 }
 
+class WindowFake {
+  readonly #eventListeners = new Map<string, () => void>()
```

Comment:
is the # just a naming convention here? no issue with it - I'm just curious as it's been cropping up a little bit lately.

Reply:
It's a private class field: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields

e.g. this is invalid:

```js
const window = new WindowFake()
window.#eventListeners // throws an error
```

Reply:
thanks!